src/transformers/models/timm_wrapper/configuration_timm_wrapper.py

```diff
@@ -80,6 +104,9 @@ def from_dict(cls, config_dict: Dict[str, Any], **kwargs):
     def to_dict(self) -> Dict[str, Any]:
         output = super().to_dict()
         output["num_classes"] = self.num_labels
+        output["label_names"] = list(self.id2label.values())
+        output.pop("id2label", None)
+        output.pop("label2id", None)
```

Comment:
For timm compatibility we store an `index to label name` mapping as a list in config